project/src/main/world/creature/creature.gd

```diff
@@ -284,12 +285,11 @@ Stores this creature's fatness in the CreatureLibrary.
 
 This allows the creature to maintain their fatness if we see them again. They lose a little weight in between.
 """
-func store_fatness() -> void:
+func save_fatness(stored_fatness: float) -> void:
```

Comment:
Document this new parameter.

Reply:
Added comment: `The fatness to save in the creature library. This can be higher...`